postgres-appliance/configure_spilo.py

```diff
@@ -111,7 +111,7 @@ def deep_update(a, b):
         wal_keep_segments: 8
         wal_log_hints: 'on'
         max_wal_senders: 5
-        shared_buffers: 500MB
+        shared_buffers: {{postgresql.parameters.shared_buffers}}
```

Comment:
I think `shared_buffers` should not be configured cluster-wide, you need to move it to 'postgresql.parameters'